content/about/funding.md

```diff
@@ -3,18 +3,18 @@ $title@: Funding
 $order: 7
 ---
 
-This page lists currently active funding for our projects
+<img style="float: right;" src="https://bit.ly/2XPyew9" width=200 height=200>
```

Comment:
styling should go in the styles.css file unless there's specific reason not to.  just give this a class name and style it in there.

we should be hosting this image the logo directly from our site, not pulling it in from github using a url shortener

Reply:
ok, sounds good

Reply:
can you delete, then we can merge the other updates in this PR to the funding page?

Reply:
Sure, done